### PR TITLE
feat(rtl): Add Support for right to left slides

### PR DESCRIPTION
**Pull request by farnabaz**
Add Support for right to left slides, with these changes dragging works fine in RTL pages, also `resistanceCoef` multiplies in right place according to rtl layout.  
I've added new props called `rtl` to support RTL direction, so this isn't break anything

### CODE REVIEW

File: src/Carousel.vue [hunk 1]
```diff
@@ -813,11 +835,14 @@ export default {
     },
     render() {
       // add extra slides depending on the momemtum speed
-      this.offset +=
-        Math.max(
-          -this.currentPerPage + 1,
-          Math.min(Math.round(this.dragMomentum), this.currentPerPage - 1)
-        ) * this.slideWidth;
+      if (this.rtl) {
```
Comment (quinnlangille) [inline src/Carousel.vue]: Not sure what's going on here - do we not want this feature with rtl scrolling (I've never seen rtl scrolling before, so this is more informative than critical lol)

Comment (farnabaz) [reply src/Carousel.vue]: My bad, I forgot to convert its logic to support RTL

Comment (quinnlangille) [reply src/Carousel.vue]: Ok cool! Glad I asked 😎

**Comment by quinnlangille** [comment]
@farnabaz thanks for the contribution! I've merged into our working branch and will be releasing asap!

**Pull request merged**